app/templates/package.json

```diff
@@ -5,8 +5,10 @@
     "backbone-torso": "0.2.1",
     "brec-tables": "0.2.5"
   },
+  "scripts": {
```

Comment:
Instead of using the npm run command, I might be able to instead spawn a command like:
$ ./node_modules/gulp/bin/gulp build